docs/docs/api/promise.config.md

```diff
@@ -26,16 +26,18 @@ Promise.config({
     // Enable long stack traces.
     longStackTraces: true,
     // Enable cancellation.
-    cancellation: true
+    cancellation: true,
+    // Enable promises monitoring.
+    monitor: true
```

Comment:
I don't think the Promise.config code was modifeid to allow this

Also monitor should be allowed to be disabled with monitor: false